test/LetConst/AssignmentToConst.js

```diff
@@ -3,29 +3,29 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-try { eval("const x = 1; x = 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x += 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x -= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x *= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x /= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x &= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x |= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x ^= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x >>= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x <<= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x >>>= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x %= 2;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x ++;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; x --;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; ++ x;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; -- x;"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; {x++;}"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; {let x = 2; x++;}"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; {x++; let x = 2;}"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; {let x = 2;} x = 10"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; {const x = 2; x++;}"); } catch (e) { WScript.Echo(e); }
-try { eval("const x = 1; {const x = 2;} x++;"); } catch (e) { WScript.Echo(e); }
-try { eval("x = 1; {let x = 2;} const x = 10;"); } catch (e) { WScript.Echo(e); }
-try { eval("function f() {x = 1; {let x = 2;} const x = 10;} f();"); } catch (e) { WScript.Echo(e); }
+try { eval("WScript.Echo('test 1'); const x = 1; x = 2;"); WScript.Echo("passed"); } catch (e) { WScript.Echo(e); }

```

Comment:
Not related to your change - but this file can easily be changed to use the UnitTestFramework, so that we don't need to dump the baseline. What do you think?